doc/rtd/tutorial/wsl.rst

```diff
@@ -0,0 +1,296 @@
+.. _tutorial_wsl:
+
+WSL Tutorial
+************
+
+In this tutorial, we will customize a Windows Subsystem for Linux (WSL)
+instance using cloud-init on Ubuntu.
+
+How to use this tutorial
+========================
+
+In this tutorial, the commands in each code block can be copied and pasted
+directly into a ``PowerShell`` Window . Omit the prompt before each
+command, or use the "copy code" button on the right-hand side of the block,
+which will copy the command for you without the prompt.
+
+Prerequisites
+=============
+
+This tutorial assumes you are running within a ``Windows 11`` or ``Windows
```

Comment:
Let's add that we are assuming wsl version 2. There are some Azure windows server offerings that have wsl  1 installed (and provide no `wsl --version`) subcommand. We can prompt folks to confirm they have version 2 installed with `wsl --version` . If the command doesn't tell you the version it's not new enough.
expected output:
```
PS > wsl --version
WSL version: 2.1.5.0
Kernel version: 5.15.146.1
WSLg version: 1.0.60
MSRDC version: 1.2.5105
Direct3D version: 1.611.1-81528511
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.20348.2402
```